Makefile

```diff
@@ -313,6 +313,17 @@ proto-api-v3-all:
 		protoc-gen-swagger/options/openapiv2.proto \
 		gogoproto/gogo.proto
 
+
+.PHONY: proto-storage-all
+proto-storage-all:
+	$(PROTOC_WITH_GRPC) \
+		proto/storage/v2/trace_storage.proto
+	$(PROTOC_WITH_GRPC) \
+		proto/storage/v2/dependency_storage.proto
+
+	$(PROTOC_INTERNAL) \
+		gogoproto/gogo.proto
```

Comment:
What is this for?

Reply:
It doesn't look like this is needed. I removed it.